pkg/virtual/framework/forwardingregistry/wrappers.go

```diff
@@ -78,3 +79,37 @@ func WithLabelSelector(labelSelectorFrom func(ctx context.Context) labels.Requir
 		return storage
 	}
 }
+
+func WithStaticLabelSelectorAndInWriteCallsCheck(labelSelector labels.Requirements) StorageWrapper {
```

Comment:
I'd rather put this new wrapper in the Syncer virtual workspace package (in builder ?)

Reply:
done